### PR TITLE
Add some tools to handle endianness

### PR DESCRIPTION
**Pull request by s-trinh**
- [Build your open source projects on IBM Power and IBM Z CPU architecture](https://blog.travis-ci.com/2019-11-12-multi-cpu-architecture-ibm-power-ibm-z)
- [Building on Multiple CPU Architectures](https://docs.travis-ci.com/user/multi-cpu-architectures/)
- [Ubuntu Xenial 16.04 as the default Travis CI build environment](https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment)
- [The Xenial Build Environment](https://docs.travis-ci.com/user/reference/xenial/)
- [Build Environment Overview](https://docs.travis-ci.com/user/reference/overview/)
- [Testing Your Project on Multiple Operating Systems](https://docs.travis-ci.com/user/multi-os/#example-multi-os-build-matrix)
- to validate the `.travis.yml`: [Travis CI Build Config Explorer](https://config.travis-ci.com/explore)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,37 @@
...
-  - gcc
-  - clang
 
-sudo: required
-dist: trusty
+jobs:
+  include:
+    - os: linux
+      dist: xenial
+      arch: amd64
```
Comment (fspindle) [inline .travis.yml]: Xenial 16.04 becomes default recently, but I'm wondering if we don't have to use bionic instead to test with 18.04.
What do you think ?

Comment (fspindle) [reply .travis.yml]: There is also the `osx_image` that could be upgraded from 10.1 to 11.3
```
osx_image: xcode11.3
```

Comment (s-trinh) [reply .travis.yml]: Yes, I have switched to Bionic and one build with Xenial.
Same thing with Xcode.

Comment (fspindle) [reply .travis.yml]: ok

**Pull request merged**